File: StressTracker5001Server/DTOs/Column/ColumnDto.cs
namespace StressTracker5001Server.DTOs.Column
{
    public class ColumnDto
    {
        public required int Id { get; set; }
        public required int BoardId { get; set; }
        public required string Name { get; set; }
        public required int Position { get; set; }
        public int? WipLimit { get; set; }
        public required DateTimeOffset CreatedAt { get; set; }
        public required DateTimeOffset UpdatedAt { get; set; }
    }
}

File: StressTracker5001App/src/dto/column.dto.ts
export interface ColumnDto {
  id: number;
  boardId: number;
  name: string;
  position: number;
  wipLimit: number | null;
  createdAt: string;
  updatedAt: string;
}

export interface ColumnCreateDto {
  name: string;
  position: number;
  wipLimit?: number | null;
}

export interface ColumnMoveDto {
  newPosition: number;
}

export interface ColumnUpdateDto {
  name?: string;
  wipLimit?: number | null;
}
